src/services/email.services.js

```diff
@@ -0,0 +1,88 @@
+import nodemailer from 'nodemailer';
+import 'dotenv/config';
+import jwt from 'jsonwebtoken';
+
+const transporter = nodemailer.createTransport({
+  host: process.env.SMTP_HOST,
+  port: process.env.SMTP_PORT,
+  auth: {
+    user: process.env.SMTP_USER,
+    pass: process.env.SMTP_PASSWORD,
+  },
+});
+
+function send({ email, subject, html }) {
+  return transporter.sendMail({
+    to: email,
+    subject,
+    html,
+  });
+}
+
+function sendActivationEmail(email, token) {
+  const href = `${process.env.CLIENT_HOST}/activation/${token}`;
+  const html = `
+  <h1>Activate account</h1>
+  <a href="${href}">${href}</a>
+  `;
+
+  return send({
+    email,
+    html,
+    subject: 'Activate',
+  });
+}
```

Comment:
Ensure that the token used in the activation email is securely generated and has an appropriate expiration time.